### DISCUSSION TITLE
Robotics Computer

### DISCUSSION THREAD

**Discussion by HannahStrimpel** [frc5567/2023-Robot#205] [Team Posts]
What all do we needed downloaded onto/done on the robotics' computer? Also, do you have links to what needs to be downloaded?

**Comment by MikasaSucasa** [comment]
We need to clone the 2021 repo so that when we go in to test the robot code, all we need to do is update to the latest master

**Comment by HannahStrimpel** [comment]
Okay. And just to double check, should I use my account to clone it or is there another account I should use?

**Comment by MikasaSucasa** [comment]
Using your account is the way to go. Let us know if you have any issues

**Comment by HannahStrimpel** [comment]
Okay

**Comment by OwenM14** [comment]
2021 Wpilib has been installed and master has been cloned onto the computer
